java-grpc/flatbuffers-java-grpc/src/test/flatbuffers/monster-service.fbs

```diff
@@ -0,0 +1,41 @@
+// Example IDL file for our monster's schema.
```

Comment:
Can this be in `grpc/tests` like the other languages so far, and use the existing schema, again like the other languages?